tasks/channels_anti_entropy_sync.py

```diff
@@ -0,0 +1,67 @@
+import discord
+import os
+import datetime
+from discord.ext import tasks
+from APIs.SweccAPI import SweccAPI
+import dotenv
+
+import logging
+
+logging.basicConfig(
+    level=logging.INFO, format="[%(asctime)s] [%(levelname)s] %(name)s: %(message)s"
+)
+
+dotenv.load_dotenv()
+
+swecc_api = SweccAPI()
+
+
+def get_channel_type(channel):
+    if isinstance(channel, discord.TextChannel):
+        return "TEXT"
+    elif isinstance(channel, discord.VoiceChannel):
+        return "VOICE"
+    elif isinstance(channel, discord.CategoryChannel):
+        return "CATEGORY"
+    elif isinstance(channel, discord.StageChannel):
+        return "STAGE"
+    elif isinstance(channel, discord.ForumChannel):
+        return "FORUM"
+    else:
+        return "UNKNOWN"
+
+
+async def sync(guild):
+    await swecc_api.sync_channels(
+        [
+            {
+                "channel_id": channel.id,
+                "channel_name": channel.name,
+                "category_id": channel.category_id,
+                "channel_type": get_channel_type(channel),
+                "guild_id": guild.id,
+            }
+            for channel in guild.channels
```

Comment:
Alternatively, if you want to keep `UNKNOWN` as a possible return value for any obscure channel types, you could avoid including channels with a type of `UNKNOWN` in this list you send to the API.